README.md

```diff
@@ -36,6 +36,20 @@ npm run dev                # Run the development server
 npm run build              # Build the entire site
 ```
 
+This node needs to be version 10.22.0, so you first have to make sure that you're entering these commands with that
```

Comment:
Perhaps it would be clearer to have "node" be "Node.js" here since it's the first time Node.js is mentioned.